sql/kringcoach_toggle.sql

```diff
@@ -0,0 +1,4 @@
+-- Refactor voor kringcoach toggle 27/10/2016
+UPDATE profielen SET kringcoach = '1' WHERE `kringcoach` IS NOT NULL;
+UPDATE profielen SET kringcoach = '0' WHERE `kringcoach` IS NULL;
+ALTER TABLE profielen CHANGE kringcoach kringcoach tinyint(1) NULL DEFAULT NULL;
```

Comment:
Begrijpelijk. Dit is vooral zodat het consistent is met de huidige praktijk (zie bijvoorbeeld de verticaleleider kolom)